### PR TITLE
New header style for application

### PR DESCRIPTION
**Pull request by willmcb**
## Context

Changes the header design to match that of the prototype.

UPDATE: following speaking to @adamsilver new header is now only used in the ProviderInterface.

## Changes proposed in this pull request

**Before**
Screenshot 2020-05-27 at 11 39 45

**After**
Screenshot 2020-05-27 at 16 44 48

## Guidance to review

- is it too reliant in MoJ imports?
- should the line under the heading match the colour of the environment tag?
- does it look okay throughout the app?
- does it collapse as you'd expect on smaller screens (I have removed the 'menu' button inline with the behaviour of the prototype.
- does this look okay if someone is signed out:
Screenshot 2020-05-27 at 16 44 14

## Link to Trello card

https://trello.com/c/st1Mq1h6/2005-build-implement-the-new-header-for-manage-applications

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by tijmenb** [comment]
> should the line under the heading match the colour of the environment tag?

Yes, definitely!

File: app/frontend/styles/_header.scss [hunk 1]
```diff
@@ -0,0 +1,170 @@
+/* ==========================================================================
```
Comment (tijmenb) [inline app/frontend/styles/_header.scss:1]: We already include the package in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/package.json#L5. Do we need to import this specifically?

Comment (willmcb) [reply app/frontend/styles/_header.scss]: Good point - will try to load it from source. In other places they are copied over, but I think it makes sense to load them from node_modules.

Comment (willmcb) [reply app/frontend/styles/_header.scss]: Done.

**Comment by adamsilver** [comment]
> does it collapse as you'd expect on smaller screens (I have removed the 'menu' button inline with the behaviour of the prototype.

Yep

> does this look okay if someone is signed out

Yep

**Comment by willmcb** [comment]
I've learnt from @adamsilver that we don't need to rely on the MoJ header styles and can instead configure the govuk styles to achieve the design we want. As this PR is becoming slightly convoluted I am going to close it and push up a separate cleaner PR.

**Pull request closed without merging**